backend/doc/apache.md

```diff
@@ -84,8 +97,8 @@ Create the file named `/etc/uwsgi/apps-available/tracim.ini` containing:
     virtualenv = [TRACIM_PATH]/backend/env
     http-socket = :6543
     socket-timeout = 360
-    #workers = 1
-    #threads = 8
+    #workers = 4
+    #threads = 4
```

Comment:
These commented value are not uncommented with keeping the value, is it normal ?

Reply:
I not understand what you says.

Reply:
This part of code is either : 
- untouched: stay uncommented
- modified with sed like  : ` sed -i "s|^#threads = .*|threads = 4|g" /etc/tracim/tracim_web.ini` 

which mean the default commented value is either overrided either commented.